Jenkins.gatekeeper

```diff
@@ -0,0 +1,122 @@
+pipeline {
```

Comment:
Is this a canary job to dig out XML changes?

Reply:
Yes, I'll remove this one...